src/js/grapl-cdk/README.md

```diff
@@ -32,9 +32,11 @@ Github.
 
 Navigate to https://github.com/grapl-security/grapl/releases and find
 the git tag associated with the latest release. Then `cd
-src/js/grapl-cdk` and run `python3 fetch_zips_by_tag.py $TAG` where
-`$TAG` is the appropriate git tag. The script will download all the
-release artifacts to the `zips/` directory.
+src/js/grapl-cdk` and run 
+`python3 fetch_zips_by_tag.py --tag $TAG --version $VERSION`
+where `$TAG` is the appropriate git tag (i.e. 'v0.1.2') 
+and `$VERSION` is the build version, like 'latest`.
```

Comment:
sidenote: i hate that we use 'v0.1.2' is the input to 'tag' and not 'version', deeply confusing

Reply:
That seems weird, and was not how it was originally used, as I recall. I think we used to have a "beta" channel/tag, and VERSION used to have the format v0.1.2.

Reply:
oh, you know what, TAG is definitely 'latest' everywhere else. I think my documentation is wrong on the --help.